autosubmit/autosubmit.py

```diff
@@ -1384,9 +1384,9 @@ def expid(description, hpc="", copy_id='', dummy=False, minimal_configuration=Fa
                 raise AutosubmitCritical(f"Experiment {copy_id} doesn't exists", 7011)
             if minimal_configuration:
                 conf_dir = Path(copy_id_folder) / "conf"
-                if not list(conf_dir.glob('minimal.y*')):
+                if Path(conf_dir, 'minimal.yml').exists() or Path(conf_dir, 'minimal.yaml').exists():
```

Comment:
My bad, I think we need to invert the logic? It was `not list()`... so it must check the files do not exist?

Reply:
Maybe that's what broke the tests? (Didn't check)